extension/persistence/relational-jdbc/src/test/java/org/apache/polaris/extension/persistence/impl/relational/jdbc/AtomicMetastoreManagerWithJdbcBasePersistenceImplTest.java

```diff
@@ -71,4 +71,11 @@ protected void testPolicyMapping() {
     // TODO: add Policy support.
     assertThrows(UnsupportedOperationException.class, super::testPolicyMapping);
   }
+
+  @Override
+  @Test
+  protected void testPolicyMappingCleanup() {
+    // TODO: add Policy support.
+    assertThrows(UnsupportedOperationException.class, super::testPolicyMappingCleanup);
+  }
```

Comment:
Lets wait, for my PR, i have something working locally with existing test on this module, should be able to publish the pr soon.

That way TODO would not be required.

Reply:
Here you go please rebase on this : https://github.com/apache/polaris/pull/1468